src/app/active-mode/Views/current-activity/VendorBounties.tsx

```diff
@@ -39,9 +40,27 @@ function mapStateToProps(state: RootState): StoreProps {
   };
 }
 
-type Props = ProvidedProps & StoreProps;
+function bountiesForActivity(bounties: VendorItem[], activity: DestinyActivityDefinition) {
+  return bounties.filter(({ item }) => {
+    const info = item?.hash && pursuitsInfo[item?.hash];
+    if (!info) {
+      return false;
+    }
+
+    for (const key in info) {
+      switch (key) {
+        case 'Destination':
+          return info[key].includes(activity.placeHash);
+        case 'ActivityMode':
+          return activity.activityModeHashes.some((modeHash) => info[key].includes(modeHash));
```

Comment:
```suggestion
          return activity.activityModeHashes?.some((modeHash) => info[key].includes(modeHash));
```

Reply:
ah thanks yeah I have a follow up that fixes some of the activity info. I'll push tomorrow